browse/services/database/__init__.py

```diff
@@ -45,8 +44,6 @@ def get_institution(ip: str) -> Optional[str]:
         return institution_name  # type: ignore
     except NoResultFound:
         return None
-    except SQLAlchemyError as e:
```

Comment:
Curious why this was removed from the DB service functions?

Reply:
Transforming the SQLAlchemyError to an IOError seems arbitrary and doesn't add anything. They both should get handled by the arxiv-base 500 handler.